### ISSUE TITLE
Rating component: In High contrast mode all stars render with the same color

### ISSUE THREAD

**Issue by pkuma-msft** [OfficeDev/office-ui-fabric-react#1829]
### Bug Report

- __Package version(s)__: (Latest)
- __Browser and OS versions__: (Edge on Windows 10)

I could not find an already existing bug on rating control in high contrast mode. The only bug I found related to accessibility was https://github.com/OfficeDev/office-ui-fabric-react/issues/1615 which talks about a different issue.

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (High) 

Products/sites affected: ()

#### Describe the issue:

Environment: Edge browser running on Windows 10 in high contrast mode
In high contrast mode the rating stars all render in the same color, so it's impossible to figure out what the rating is visually

#### Actual behavior:
**Repro**
1. Open Edge on Windows 10 and press left-Shift + left-Alt + PrtScreen to enable high contrast mode
2. Go to https://dev.office.com/fabric#/components/rating
3. Select the stars in rating control and observe that there is no visual difference between selected and non-selected stars
4. Toggle out of high contrast mode by pressing left-Shift + left-Alt + PrtScreen

This is how the stars render,
![image](https://cloud.githubusercontent.com/assets/13727302/26293011/3c0513a6-3ed8-11e7-8947-870d312b661b.png)

This is how they look in normal mode,
![image](https://cloud.githubusercontent.com/assets/13727302/26293024/507c9228-3ed8-11e7-8919-d4bfb15af712.png)

#### Expected behavior:

In high contrast mode, selected stars should be visually different than non-selected ones.
